frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -75,6 +75,26 @@ import { isBoundingBoxExportable } from "./download_modal_view";
 
 const { ThinSpace } = Unicode;
 
+type MinimalBoundingBoxExtent = {
+  minExtent: Vector3;
+  mag: Vector3;
+};
+
+// The following minimal bounding box extents are based on the default model that is used for neuron and mitochondria segmentation.
+// Thus when changing the default model, consider changing these values as well.
+// See https://github.com/scalableminds/webknossos/issues/8198#issuecomment-2782684436
+const MIN_BBOX_EXTENT_FOR_NEURON_SEGMENTATION: MinimalBoundingBoxExtent = {
+  minExtent: [16, 16, 4],
+  mag: [1, 1, 1],
+};
+
+const MIN_BBOX_EXTENT_FOR_MITO_SEGMENTATION: MinimalBoundingBoxExtent = {
+  minExtent: [16, 16, 4],
```

Comment:
@AndreaNathansen Do you know what the minimum bounding box and or dataset size for the mito model in wk is? See https://github.com/scalableminds/webknossos/issues/8198 for context

Reply:
I don't know exactly, would have to try it out.

Since it predicts slice-wise, maybe one slice in z direction would be sufficient, but this could fail during the median pooling afterwards.

For x and y direction, the MitoNet paper says "the output from the encoder was 16x smaller than the input", so maybe 16 vx is the limit here, but that would have to be tried as well.

Reply:
is there a way to find out the voxel sizes of the finest available mag of the training dataset? i think that would need to be hardcoded as well, right? or is this a constant across all datasets? I found the scale 11.24 × 11.24 × 28 nm³/voxel in multiple datasets but that could be a coincidence, didnt find anything explicit in the docs

Reply:
Sorry, my statement was ambiguous. I think you don't need to know the voxel size of the finest available mag of the training dataset. Contrary to the neuron inference (where the mag of the current dataset that most closely resembles the mag the model was trained in, is chosen for inference), for the mito inference, the finest available mag _of the current dataset_ is chosen for inference. So there is no need to do any mag selection according to the mean training voxel size.

Reply:
oh I see, thank you :D

Reply:
@AndreaNathansen can you find this out or is there anything I can do?

Reply:
thank you for finding this out! actually both is relevant IMO, I am asserting that both the bounding box of the segmentation and the overall dataset are big enough. for nuclei and neuron segmentation I am asserting that the DS is double the size of the minimal bounding box